devices.js

```diff
@@ -9140,6 +9140,20 @@ const devices = [
         },
         exposes: [e.cover_position().setAccess('state', ea.ALL)],
     },
+    {
+        zigbeeModel: ['SSW-S1'],
```

Comment:
According to your database this should be `WSS-4E_00.00.03.08TC`. Does it really work with the current config?

Reply:
Bad copy&paste, misplaced zigbeeModel and model.
`    {
        zigbeeModel: ['WSS-4E_00.00.03.08TC'],
        model: 'SSW-S1',
        vendor: 'Blaupunkt',
        description: '4-Touch single click buttons',
        fromZigbee: [fz.ignore_power_report, fz.command_recall, fz.legacy.AV2010_34_click],
        toZigbee: [],
        exposes: [e.action(['recall_*'])],
        meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
        },
    },`